backend/canvas_app_explorer/migrations/0005_adding_mysql_cache.py

```diff
@@ -0,0 +1,25 @@
+from django.db import migrations
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        # Add a dependency in here on an existing migration in the app you
+        # put this migration in, for example:
+        ('canvas_app_explorer', '0004_auto_20211103_1846'),
+    ]
+
+    operations = [
+        migrations.RunSQL(
+            """
```

Comment:
It was autogenerated from ` ./manage.py mysql_cache_migration`. This was the format the library created. 

I'm guessing it's because they set specific character sets (for some reason) and if you just did a regular migration it would create it as the default character set. 

https://django-mysql.readthedocs.io/en/latest/cache.html#usage